__tests__/DateBlock.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { DateBlock } from '../components';
+
+describe('Component: DateBlock', () => {
+  const props = {};
+
+  beforeEach(() => {
+    props.date = '2015-03-14T12:00:00Z';
+  });
+
+  it('renders without crashing', () => {
+    const wrapper = shallow(<DateBlock {...props} />);
+    expect(wrapper).toHaveLength(1);
```

Comment:
i'm not familiar with jest. I'm a bit overwhelmed with the tooling in this project so I'm just trying to plow ahead with tests, but if it provides a good return on investment I'm open to suggestions